examples/resources/spectrocloud_cluster_aws/resource.tf

```diff
@@ -69,14 +70,26 @@ resource "spectrocloud_cluster_aws" "cluster" {
     count                   = 1
     instance_type           = "t3.large"
     disk_size_gb            = 62
-    azs                     = ["us-west-2a"]
+    azs                     = ["us-east-2a"]
```

Comment:
`azs` and `az_subnets` are mutually exclusive. An example should have `azs` for dynamic provisioning and `az_subnets` for static provisioning